### PR TITLE
Add a command line version of Package.install

### PR DESCRIPTION
**Pull request by dimaryaz**

### CODE REVIEW

File: api/python/quilt3/main.py [hunk 1]
```diff
@@ -46,6 +46,40 @@ def create_parser():
     )
     config_p.set_defaults(func=cmd_config)
 
+    shorthelp = "Install a package"
+    install_p = subparsers.add_parser("install", description=shorthelp, help=shorthelp)
+    install_p.add_argument(
+        "name",
+        help="Name of package, in the user/pkg format",
```
Comment (akarve) [inline api/python/quilt3/main.py]: In docs we usually use all caps for template strings.
```suggestion
        help="Name of package, in the USER/PKG format",
```

File: api/python/quilt3/main.py [hunk 2]
```diff
@@ -46,6 +46,40 @@ def create_parser():
...
+        type=str,
+    )
+    install_p.add_argument(
+        "--registry",
+        help="Registry where package is located. Defaults to the default remote registry.",
```
Comment (akarve) [inline api/python/quilt3/main.py]:
```suggestion
        help="Registry where package is located, usually s3://MY-BUCKET). Defaults to the default remote registry.",
```

Comment (armandmcqueen) [inline api/python/quilt3/main.py]:
```suggestion
        help="Registry where package is located, usually s3://MY-BUCKET.",
```

File: api/python/quilt3/main.py [hunk 3]
```diff
@@ -46,6 +46,40 @@ def create_parser():
...
+        type=str,
+        required=False,
+    )
+    install_p.add_argument(
+        "--dest",
+        help="Local path to download files to.",
+        type=str,
+        required=False,
+    )
+    install_p.add_argument(
```
Comment (armandmcqueen) [inline api/python/quilt3/main.py]: Do we really need this argument? Having both destination and destination registry seems confusing to me.

File: api/python/quilt3/main.py [hunk 4]
```diff
@@ -46,6 +46,40 @@ def create_parser():
...
+        "--top-hash",
+        help="Hash of package to install. Defaults to latest.",
+        type=str,
+        required=False,
+    )
+    install_p.add_argument(
+        "--dest",
```
Comment (akarve) [inline api/python/quilt3/main.py]: What we call `--dest` is probably better called `--prefix`, but not change for now

File: api/python/quilt3/main.py (hunk 3 above)
Comment (dimaryaz) [reply api/python/quilt3/main.py]: True... But we should fix the Python API

**Pull request merged**